### DISCUSSION TITLE
Consistent footer between the main site and the documentation

### DISCUSSION THREAD

**Discussion by tbogosavljevic** [semaphoreci/semaphore#270] [Documentation improvements]
After a discussion with @mateuscap, we noticed that our documentation seemingly lacked any links to our status page. I discovered that the footer of semaphoreci.com includes those, while our docs do not.

I suggest that in the aim of thematic consistency between these two sites, we incorporate the same footer as we have on semaphoreci.com. I understand that this may not be a quick thing, and also falls under the topic of making the thematic language between the main site and the doc site the same (as defined in [this task](https://github.com/orgs/semaphoreci/projects/2?pane=issue&itemId=66939068&issue=semaphoreci%7Csemaphore%7C55)). I am opening this discussion for anyone to share their thoughts about this!

**Comment by TomFern** [comment]
The quick solution would be to add the link to the status page, the links are all defined on docusaurus.config.js, so it's an easy fix. As to cosmetic changes, all ideas are welcome. I'm thinking that whatever changes we do need to work on light and dark themes (something that semaphoreci.com does not have).
